AIPscan/API/namespace_data.py

```diff
@@ -114,3 +114,43 @@ def get(self, storage_service_id, file_type=None, limit=20):
             storage_service_id=storage_service_id, file_type=file_type, limit=limit
         )
         return file_data
+
+
+@api.route("/storage_service/<storage_service_id>/file-format")
+class AIPsByFormatList(Resource):
+    @api.doc(
+        "list_aips_by_format",
+        params={
+            "file_format": {
+                "description": "File format name (must be exact match)",
```

Comment:
I completely agree that an additional API endpoint that lists the file format names would be very useful here. I think I've been GUI-centric in my thinking because of the immediate priorities, so I didn't think about including that additional API endpoint in the scope of this PR. But I'm happy to revisit that.

On the endpoints, see my comments in the main thread.